cypress/e2e/data-import/importing-marc-bib-files/import-edit-derive-bib-with-minimum-fields.cy.js

```diff
@@ -0,0 +1,93 @@
+    expectedFieldsInQuickMarc: ['LDR', '001', '008', '005', '245', '999'],
+    marcFile: {
+      marc: 'MARC_Bib_bare_minimum_C387435.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create instance and SRS MARC Bib'
+    }
+  };
+
+  const createdRecordIDs = [];
+
+  before('Creating data', () => {
+    cy.createTempUser([
+      Permissions.inventoryAll.gui,
+      Permissions.moduleDataImportEnabled.gui,
+      Permissions.uiQuickMarcQuickMarcBibliographicEditorAll.gui,
+      Permissions.uiQuickMarcQuickMarcEditorDuplicate.gui
+    ]).then(createdUserProperties => {
+      testData.userProperties = createdUserProperties;
+      cy.login(testData.userProperties.username, testData.userProperties.password, { path: TopMenu.dataImportPath, waiter: DataImport.waitLoading });
+    });
+  });
+
+  after('Deleting created user and data', () => {
+    Users.deleteViaApi(testData.userProperties.userId);
+    createdRecordIDs.forEach(recordID => {
+      InventoryInstance.deleteInstanceViaApi(recordID);
+    });
+    cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading });
+    DataImport.selectLog();
+    DataImport.openDeleteImportLogsModal();
+    DataImport.confirmDeleteImportLogs();
+  });
+
+  it('C387435 Import and edit/derive "MARC Bib" record having only required fields (spitfire)', { tags: [TestTypes.criticalPath, DevTeams.spitfire] }, () => {
+    DataImport.uploadFile(testData.marcFile.marc, testData.marcFile.fileName);
+    JobProfiles.waitLoadingList();
+    JobProfiles.searchJobProfileForImport(testData.marcFile.jobProfileToRun);
+    JobProfiles.runImportFile();
+    JobProfiles.waitFileIsImported(testData.marcFile.fileName);
+    Logs.checkStatusOfJobProfile(JOB_STATUS_NAMES.COMPLETED);
+    Logs.openFileDetails(testData.marcFile.fileName);
+    Logs.getCreatedItemsID().then(link => {
+      createdRecordIDs.push(link.split('/')[5]);
+      Logs.goToTitleLink('Created');
+      InventoryInstance.checkInstanceTitle(testData.initialTitle);
+
+      InventoryInstance.editMarcBibliographicRecord();
+      QuickMarcEditor.checkFieldsExist(testData.expectedFieldsInQuickMarc);
+      QuickMarcEditor.checkFieldsCount(testData.expectedFieldsInQuickMarc.length);
+      inventorySteps.verifyHiddenFieldValueIn008(createdRecordIDs[0], 'Entered', testData.initial008EnteredValue);
+      QuickMarcEditor.updateExistingField('245', '$a ' + testData.updatedTitleEdit);
+      QuickMarcEditor.pressSaveAndClose();
+      QuickMarcEditor.checkAfterSaveAndClose();
+
+      InventoryInstance.deriveNewMarcBib();
+      QuickMarcEditor.checkFieldsExist(testData.expectedFieldsInQuickMarc);
+      QuickMarcEditor.checkFieldsCount(testData.expectedFieldsInQuickMarc.length);
+      QuickMarcEditor.checkContent('$a ' + testData.updatedTitleEdit, 4);
+      QuickMarcEditor.updateExistingField('245', testData.updatedTitleDerive);
+      QuickMarcEditor.pressSaveAndClose();
```

Comment:
please use click instead of press in methods name

Reply:
This method was not added by me. And I do not want to change it's name as it is used in a lot of other tests

Reply:
ok, I'll change it